third_party/move/move-compiler-v2/src/function_checker.rs

```diff
@@ -89,156 +93,175 @@ pub fn check_access_and_use(env: &mut GlobalEnv) {
                     },
```

Comment:
Why do we need to record `private_funcs` if `!before_inlining`? (applies to the code above where we add to `private_funcs`, but I could not comment on those lines as they were not changed).